### ISSUE TITLE
UI of nested field get weird after pushed "Add a new ..." button

### ISSUE THREAD

**Issue by ha4gu** [railsadminteam/rails_admin#3528]
**Describe the bug**

The UI of nested field for `has_many` association won't work properly on Rails Admin 3.0.0.
After pushed "Add a new ..." button, you would find two strange points:

- The width of instance tabs broadly extends, as wide as other input field.
- The nested field won't be closed; it would show up just after toggled up.

These behaviors occur in these cases:

- Create (new) view

![new-failed](https://user-images.githubusercontent.com/9391978/172089864-459ff583-aca0-4249-9bbc-c3d3d51689f2.gif)

- Update (edit) view, if you push add button without opening the field beforehand

![edit-failed](https://user-images.githubusercontent.com/9391978/172089879-4aeefc2c-1d69-4a7c-86b9-9422d2c6fb57.gif)

Not occur (works properly) in this case:

- Update (edit) view, if you push add button after opening the field at first

![edit-ok](https://user-images.githubusercontent.com/9391978/172089936-b692498f-5081-4170-b6ea-c260e4ffd369.gif)

**Reproduction steps**

It's easy to reproduce this behavior; Simply create two models connected with `has_many/belongs_to` association.
Here is my examples.

`app/models/author.rb`

```ruby
class Author < ApplicationRecord
  has_many :books, dependent: :destroy
  accepts_nested_attributes_for :books
end
```

`app/models/book.rb`

```ruby
class Book < ApplicationRecord
  belongs_to :author
end
```

`config/initializers/rails_admin.rb`

```ruby
RailsAdmin.config do |config|
  config.asset_source = :webpacker

  config.actions do
    ((snip))
  end

  config.included_models = ['Author', 'Book']

  config.model 'Book' do
    include_fields :title
  end
end
```

**Expected behavior**

In the UI of nested field,

- The width of instance tabs is fit to the length of strings inside, not fully expanded.
- The nested field would be opened and then closed when toggle button is switched.

**Additional context**

- `rails` version: 6.1.6
- `rails_admin` version: 3.0.0, and also current master (652418c)
- `rails_admin` npm package version: 3.0.0, and also current master (652418c)

**Comment by mshibuya** [comment]
Thanks for reporting, fixed by d1f115425c4a8d119fdeb37802fe472ae278918d.

**Comment by ha4gu** [comment]
I appreciate it! Now I confirmed the current master branch nicely resolved this issue. Thank you so much.
